dev/AppLifecycle/AppInstance.cpp

```diff
@@ -358,13 +358,14 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
             featureUsageReported = true;
         }
 
-        // TODO: Redirect UWP requests to CoreApplication::RequestRestartAsync().
-
-        // For packaged, redirect to the already existing API.
-        /*if (IsPackagedProcess())
+        // If a better way of detecting UWP is created in the future, this check should change.
+        if (IsPackagedProcess() && wil::get_token_is_app_container())
```

Comment:
AppContainer and Packaged Process are not UWP specific.  https://scorpiosoftware.net/2019/01/15/fun-with-appcontainers/

Whereas `CoreWindow` is.